contrib/freebsd/pkg-plist

```diff
@@ -0,0 +1,189 @@
+bin/fwupdagent
+bin/fwupdmgr
+bin/fwupdtool
+%%ETCDIR%%/daemon.conf
+%%ETCDIR%%/remotes.d/lvfs-testing.conf
+%%ETCDIR%%/remotes.d/lvfs.conf
+%%ETCDIR%%/remotes.d/vendor-directory.conf
+%%ETCDIR%%/remotes.d/vendor.conf
+etc/pki/fwupd-metadata/GPG-KEY-Linux-Foundation-Metadata
+etc/pki/fwupd-metadata/GPG-KEY-Linux-Vendor-Firmware-Service
+etc/pki/fwupd-metadata/LVFS-CA.pem
+etc/pki/fwupd/GPG-KEY-Linux-Foundation-Firmware
+etc/pki/fwupd/GPG-KEY-Linux-Vendor-Firmware-Service
+etc/pki/fwupd/LVFS-CA.pem
+include/fwupd-1/fwupd.h
```

Comment:
Any chance you can use wildcards in this file?

Reply:
The file is not required to build the package (if there is no pkg-plist, the list will be generated during the build).